package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@pmedianetwork/eslint-plugin",
+    "name": "@adverity/eslint-plugin",
```

Comment:
is this going to affect current consumers?
this is need to be a breaking change?

Reply:
I think it can cause issues. Lets not change it for now please.

We are freezing this repo for now btw. I don't think we will have critical issues regarding this in next week or so :) 
We started an analysis of the migration issue.
https://github.com/pmedianetwork/adverity-presense-frontend/issues/911

We will work on the release once the code has been moved and you are back from :palm_tree:

Reply:
okay.. well, I still manually published 5.1.0 and 5.1.1 to the private registry.

Reply:
https://github.com/pmedianetwork/eslint-config/pull/95/commits/e0d8ee4e21669087e8b77ca31be8e4408711007e